src/redux/reducers/index.js

```diff
@@ -1,21 +1,6 @@
-import actionTypes from "../actionTypes";
-
-const { ADD_ARTICLE } = actionTypes;
-
-const initialState = {
-  counter: 0,
-  articles: [{ title: "hello world", id: 0, body: "Hello world", userId: 101 }]
-};
-
+const initialState = {};
+/* eslint-disable no-unused-vars */
 const rootReducer = (state = initialState, action) => {
-  if (action.type === ADD_ARTICLE) {
-    return Object.assign({}, state, {
-      counter: state.counter + 10,
-      articles: state.articles.concat(
-        action.payload.slice(state.counter + 1, state.counter + 11)
-      )
-    });
-  }
   return state;
 };
 
```

Comment:
Why are we disabling no unused vars?

Reply:
It is because the file is empty since I removed @YvesIraguha's dummy components. So, the file was claiming the unused var `actions` and decided to disable it temporarily as there will be some changes in the near future that will use the `action` var in that file.

Reply:
Good